x-pack/plugins/ml/public/application/components/data_grid/data_grid.tsx

```diff
@@ -291,6 +292,24 @@ export const DataGrid: FC<Props> = memo(
             <EuiSpacer size="m" />
           </div>
         )}
+        {ccsWarning && (
+          <div data-test-subj={`${dataTestSubj} ccsWarning`}>
+            <EuiCallOut
+              title={i18n.translate('xpack.ml.dataGrid.CcsWarningCalloutTitle', {
+                defaultMessage: 'Cross Cluster Search returned no fields data.',
+              })}
+              color="warning"
+            >
+              <p>
+                {i18n.translate('xpack.ml.dataGrid.CcsWarningCalloutBody', {
+                  defaultMessage:
+                    'There was an issue retrieving data for the index pattern. Source preview in combination with Cross Cluster Search is only supported for versions 7.10 and above. You may still configure and create the transform.',
```

Comment:
As above, I think this should be `cross-cluster search`.

Reply:
Fixed in fc4e950.